components/transfer/commons/transfer_list.lua

```diff
@@ -203,6 +204,34 @@ function TransferList:fetch()
 		Array.appendWith(groupedData, currentGroup)
 	end)
 
+	local containsCoach = function(str)
+		return string.lower(str or ''):find('coach') ~= nil
```

Comment:
IMO this should apply to all staff roles, which likely makes it easier to check whether someone is a player, by looking for empty role or one of the special roles

Reply:
what special roles?
there are quite a few across the wikis ...

Reply:
I think the order could look something like this:
-no role
-loan
-inactive
-other roles

Reply:
there is also the issue with some wikis (e.g. lol) setting it in the icon stuff
and that also leads to their in game roles having to be excluded
would probably need a complete list of those across all wikis ...

Reply:
Honestly i'd argue to ignore the position field for now then

Reply:
in that case the ones that requested this will not get the change XD

Reply:
Perhaps we should do some cleanup to have them use roles if they mean roles, and have position be reserved for ingame positions